src/kata6.joinNames.js

```diff
@@ -1,3 +1,12 @@
-const joinNames = (namesObj) => {};
+const joinNames = (namesObj) => {
+  
+const nameArray = namesObj.map((item) => item.name);
+   if (nameArray.length <= 1) {
```

Comment:
Just indentation is out here. See comment on the getEmployerRole test for what you could do to sort this all out in one go!